mpas_analysis/ocean/climatology_map_sose_temperature.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, config, mpasClimatologyTask,
                         refFieldName=refFieldName,
                         refTitleLabel=refTitleLabel,
                         diffTitleLabel=diffTitleLabel,
-                        unitsLabel=r'$^\circ$C',
+                        unitsLabel=r'$^o$C',
```

Comment:
@gstreletz why did you make this chance?  The original is the standard way of getting a degree sign.

Reply:
For uniformity with the way it's done in climatology_map_sst.py.  I agree that \circ is the standard way, and that 's what I would do in LaTeX.  I went with ^o because in the case of the Matplotlib titles and axis labels, I though that it looked better (\circ is really small for some reason, which leads me to suspect that LaTeX isn't being used for the rending in Matplotlib).  We can go with either one, though, and I can make things consistent between climatology_map_{SST,sose_temperature,argo}.py.

Reply:
Ah, gotcha!  Still I'd prefer you restrict your changes to only things necessary for transects, and not worry about general coherence of the other tasks.  If you want, you could make a different pull request to do this kind of clean up.

Regarding the degree symbol, I'll see if I can find a better way.  Now that you point it out, I agree matplotlib is doing something wrong.  You are correct, it's not using LaTex (and we don't want it to since everything else looks dumb when it does).